packages/gatsby/src/utils/get-ssl-cert.js

```diff
@@ -3,6 +3,14 @@ const report = require(`gatsby-cli/lib/reporter`)
 const fs = require(`fs`)
 const path = require(`path`)
 
+const resolvePath = (directory, filePath) => {
+  // Support absolute paths
+  if (filePath[0] === `/`) {
```

Comment:
How about [path.isAbsolute](https://nodejs.org/api/path.html#path_path_isabsolute_path)?

```suggestion
  if (path.isAbsolute(filePath[0])) {
```

Also why just one part of the path? I'm a little confused by that.

Reply:
`filePath` is a string, so I was looking at the first character. I'll adjust it to use the path method